2_base_and_sensor.ipynb

```diff
@@ -436,135 +812,259 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "Sensor values are stored using Python \"data structure\".\n",
+    "\n",
+    "The contents of the data structure can be accessed via \"[variable name].[structure element name]\"."
```

Comment:
If you are referring to a specific data structure that was specified before, then:

    "The contents of the data structure can be accessed via \"[variable name].[structure element name]\"."

However, if this is a general statement that is meant to apply to any one of a number of data structures, then:

    "The contents of a data structure can be accessed via \"[variable name].[structure element name]\"."